stations.csv

```diff
@@ -70491,4 +70491,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 74632;Brescia Violino;brescia-violino;;;45.54553750281283;10.168658833685743;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;8317451;t;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74632;
 74633;Minusio;minusio;;;46.17404650703084;8.819574802968333;;CH;Europe/Zurich;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;8505417;t;;f;;f;;f;8505417;t;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74633;
 74634;Catania (Piazza Risorgimento);catania-piazza-risorgimento;;;37.50155099221305;15.068981718421918;22160;IT;Europe/Rome;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74634;
-74635;Monsampolo del T;monsampolo-del-t;;;42.881404;13.803619;19616;IT;Europe/Rome;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;S07023;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74635;
\ No newline at end of file
+74635;Monsampolo del T;monsampolo-del-t;;;42.881404;13.803619;19616;IT;Europe/Rome;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;S07023;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74635;
+74636;Bitritto;bitritto;;;41.04419684107557;16.832738826004018;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;8311252;t;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74636;
+74637;Bari Loseto;bari-loseto;;;41.04398202300499;16.8472726278579;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;8311251;t;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74637;
+74638;Bari S. Rita;bari-s-rita;;;41.07330246992781;16.851540354841326;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;8311250;t;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74638;
```

Comment:
Can you please use the full name Bari Santa Rita?
 https://it.wikipedia.org/wiki/Stazione_di_Bari_Santa_Rita

Reply:
Done